packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxUserMenu/OnyxUserMenu.vue

```diff
@@ -60,11 +61,16 @@ const isMobile = inject(
     </template>
 
     <template #options>
-      <slot></slot>
+      <div class="onyx-user-menu__options">
+        <slot></slot>
+      </div>
     </template>
 
     <template v-if="slots.footer" #footer>
-      <div class="onyx-user-menu__footer onyx-text--small">
+      <OnyxListItem v-if="isMobile" class="onyx-user-menu__mobile-footer" disabled>
```

Comment:
Would prefer to do this in the UserMenuLayout that is responsible for switching mobile/desktop layout

Reply:
done